### PR TITLE
DHE-783 Release checklist

### PR DESCRIPTION
**Pull request by sdonk**

### CODE REVIEW

File: release-checklist.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Release checklist
+=================
+
+1. Release branch created off develop at a given commit
+2. Confirm CI is green on release branch
+3. Confirm users have been made aware of scheduled downtime
+4. Open PR release branch -> master
```
Comment (jim68000) [inline release-checklist.md]: (if required)

File: release-checklist.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+5. Merge release branch into master and tag it
+6. Make sure someone is with you
+7. Stop the celery workers
+8. Trigger Jenkins job to release
+9. Apply migrations manually (if needed)
+10. If release is successful delete release branch
```
Comment (jim68000) [inline release-checklist.md]: Do you need to restart the celery workers as a final step?

Comment (sdonk) [inline release-checklist.md]: That is a very good point indeed

**Comment by sdonk** [comment]
@Singletoned @jim68000 changes made, please have a look

**Pull request merged**